src/css/custom.css

```diff
@@ -6,31 +6,31 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #0c6a02;
-  --ifm-color-primary-dark: #0b5f02;
-  --ifm-color-primary-darker: #0a5a02;
-  --ifm-color-primary-darkest: #084a01;
-  --ifm-color-primary-light: #0d7502;
-  --ifm-color-primary-lighter: #0e7d02;
-  --ifm-color-primary-lightest: #108a03;
+  --ifm-color-primary: #a000c8;
```

Comment:
I find the green color much more comfortable. Should we take a vote?

![image](https://user-images.githubusercontent.com/71560916/202155963-bec0b7a3-2b10-4eca-9b90-dbe45bee62e7.png)

![image](https://user-images.githubusercontent.com/71560916/202156003-27e3aec0-e10f-44a9-a262-1b9093925fc7.png)

Reply:
Im OK with a vote. @jboileau99 @TheKinrar what do you prefer?

Feel free to directly commit whatever emerges as preference.

Reply:
Third option is to use the [default theme](https://docusaurus.io/docs/configuration), which I prefer the most.

Reply:
I also prefer the default theme over the other two

Reply:
I updated the css with the default colors